README.md

```diff
@@ -1 +1,231 @@
-# budget-app
\ No newline at end of file
+<div align="center">
+  <h3><b>Budget App</b></h3>
+</div>
+
+![mobile (3)](https://user-images.githubusercontent.com/105460631/220934675-15f971f4-a699-4f19-9905-8fe55d204725.png)
+![mobile (4)](https://user-images.githubusercontent.com/105460631/220934738-f42cc3e1-a546-46c1-a46f-955a4863aba1.png)
+![mobile (5)](https://user-images.githubusercontent.com/105460631/220934762-99f43feb-0e15-491c-8f29-8659fc798727.png)
+
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Budget App <a name="about-project"></a>
+
+E-neza App is a frinchised online web application that allows the user to manage his/her budget: they have a list of transactions associated with a category, so that the user can see how much money they spend and on a particular category.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://ruby-lang.org/">Ruby</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://ruby-lang.org/">Rails Default Server</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="Live-presentation"></a>
```

Comment:
- [ ] As requested by the previous reviewer please deploy your App 😉

Reply:
@hiromataba Thank you for your review.